tests/test_datacard.py

```diff
@@ -18,14 +18,48 @@
 BUILD = audeer.path('..', 'build', 'html')
 
 
-def test_datacard_example(db, cache):
+@pytest.mark.parametrize(
+    'db',
+    [
+        'minimal_db',
+        'medium_db',
+    ],
+)
+def test_datacard(db, cache, request):
+    """Test datacard creation from jinja2 templates."""
+    db = request.getfixturevalue(db)
+    dataset = audbcards.Dataset(db.name, pytest.VERSION, cache)
```

Comment:
Is there a reason for having an asymmetric implementation, i.e. handling `name` in the fixture and `version` in `conftest.py`?

Reply:
You have a valid point here.

We could add the version as a parameter to the fixture and publish the database under that version.
If we publish databases with multiple versions, it would just publish the same database for all the different versions. Don't know if this will be sufficient, or if we need to also ensure differences between the different versions. Then it will not really work by using version as a parameter of the fixture.

An alternative would be that the fixture returns the version number, then we would have something like:

```python
db, version = request.getfixturevalue(db)
```

and could encode the database version in the name of the fixture if we need different versions of the same database.

Any opinion?